packages/manager/modules/pci/src/projects/project/data-processing/submit-job/spark-config/spark-config.controller.js

```diff
@@ -50,6 +55,30 @@ export default class {
       true,
     );
 
+    this.units = [
+      {
+        name: 'minutes',
+        description: this.$translate.instant(
+          'data_processing_submit_job_stepper_spark_select_unit_minutes',
+        ),
+      },
+      {
+        name: 'hours',
```

Comment:
Aren't you missing multipliers here ?

Reply:
Added: https://github.com/ovh/manager/pull/7069/commits/672e694c8562d0c4de36b0008c6ac93affa8a369